src/EditorFeatures/CSharpTest/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -1028,5 +1028,57 @@ void Error()
 }",
 CodeStyleOptions.QualifyMethodAccess);
         }
+
+        [WorkItem(17711, "https://github.com/dotnet/roslyn/issues/17711")]
```

Comment:
Addressed in commit 062c7f9.

I think all comments from your review are addressed now, but I'm not sure how to indicate this on github (other than through a comment like this one).